conda_index/index/convert_cache.py

```diff
@@ -159,7 +159,7 @@ def basename(path):
             return path
         return path.rsplit("/")[-1]
 
-    conn.create_function("migrate_basename", narg=1, func=basename, deterministic=True)
+    conn.create_function("migrate_basename", 1, basename, deterministic=True)
```

Comment:
Python developers feel that this is cleaner.
https://docs.python.org/3.15/whatsnew/3.15.html#id2